Makefile.custom.mk

```
@@ -54,6 +54,7 @@ $(KUSTOMIZE): ## Download kustomize locally if necessary.
 
 $(YQ): ## Download yq locally if necessary.
 	@$(call say,Download yq)
+	mkdir ./bin || true
```

```suggestion
	mkdir ./bin || :
```

Just in case you didn't know this variant, yet. 😉

---

thanks, useful to know but definitely not as clear as to what it's doing - i prefer readability over saving 3 characters ;)

---

Yeah, assumed that. I think I read the colon way somewhere else once and was wondering what it's doing.